capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -1,9 +1,46 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { WikiSearchResult } from '../WikiSearchTemplate';
+import { WikiResultsService } from '../wiki-results.service';
+import { mergeMap, switchMap, map, takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
 
 @Component({
-  selector: 'app-wiki-media',
+  selector: 'app-wiki-media-history-section',
   templateUrl: './wiki-media.component.html',
   styleUrls: ['./wiki-media.component.scss']
 })
-export class WikiMediaComponent{
+export class WikiMediaComponent implements OnInit, OnDestroy {
```

Comment:
This component should have a test. Something like: 

```
it('displays location history', () => {
...
}
```

Reply:
Is that something that should be a TODO for another PR?

Reply:
No, tests should usually go in with the same PR. Otherwise other contributors might break something before tests get added.

Reply:
Makes sense, I hadn't thought of that.